### PR TITLE
Wrong network modal for veSDL page

### PR DESCRIPTION
**Pull request by nkrulikovsky**
## What does this PR do?

- Display wrong network modal for non-mainnet.
- Fix translation of total veSDL

## Context

closes: https://github.com/saddle-finance/saddle-frontend/issues/1068

## Screenshot

image

### CODE REVIEW

File: public/locales/en.json [hunk 1]
```diff
@@ -293,6 +294,7 @@
   "utilization": "Utilization",
   "veSdl": "veSDL",
   "veSdlCalculator": "veSDL calculator",
+  "veSdlNetworkText": "Seems like you are in {{chainName}} network.\n veSDL is available on mainnet! \n Please switch to Ethereum mainnet.",
```
Comment (nguyenaiden) [inline public/locales/en.json:297]: nit: I'd change it to either:
Looks like you're on the {{chainName}} network. veSDL is only available on the Ethereum mainnet!
or 
Looks like you're on an unsupported network. Please switch to the Ethereum Mainnet!

Comment (nkrulikovsky) [reply public/locales/en.json:297]: Thank you for giving me advice 🙏🏻 . I wanted to ask about it.

File: public/locales/en.json [hunk 2]
```diff
@@ -293,6 +294,7 @@
   "utilization": "Utilization",
   "veSdl": "veSDL",
   "veSdlCalculator": "veSDL calculator",
+  "veSdlNetworkText": "Looks like you are in {{chainName}} network.\n veSDL is only available on the Ethereum Mainnet!",
```
Comment (hammeiam) [inline public/locales/en.json:297]: are we using Sdl or SDL for keys?

**Review by hammeiam — APPROVED** [review]

File: public/locales/en.json (hunk 2 above)
Comment (nkrulikovsky) [reply public/locales/en.json:297]: I think SDL will be good. But considering camel case, I used `veSdl` and this problem is what I wanted to ask before.
Will I change all `veSdl` to `veSDL`?

Comment (nkrulikovsky) [reply public/locales/en.json:297]: Because our UI is using veSDL, so I recommend to use veSDL  than veSdl.

File: public/locales/en.json [hunk 3]
```diff
@@ -3,7 +3,11 @@
   "approveSpend": "Approve {{tokenName}} spend",
   "available": "Available",
   "avgLockTime": "Avg. lock time",
+<<<<<<< HEAD
```
Comment (ug02fast) [inline public/locales/en.json:6]: delete

**Review by ug02fast — CHANGES_REQUESTED** [review]

**Review by ug02fast — APPROVED** [review]

**Pull request merged**